pl_bolts/models/self_supervised/swav/transforms.py

```diff
@@ -0,0 +1,200 @@
+        self.transform.append(online_train_transform)
+
+    def __call__(self, sample):
+        multi_crops = list(
+            map(lambda transform: transform(sample), self.transform)
+        )
+
+        return multi_crops
+
+
+class SwAVEvalDataTransform(SwAVTrainDataTransform):
+    def __init__(
+        self,
+        normalize=None,
+        size_crops: List[int] = [96, 36],
+        nmb_crops: List[int] = [2, 4],
+        min_scale_crops: List[float] = [0.33, 0.10],
+        max_scale_crops: List[float] = [1, 0.33],
+        gaussian_blur: bool = True,
+        jitter_strength: float = 1.
+    ):
+        super().__init__(
+            normalize=normalize,
+            size_crops=size_crops,
+            nmb_crops=nmb_crops,
+            min_scale_crops=min_scale_crops,
+            max_scale_crops=max_scale_crops,
+            gaussian_blur=gaussian_blur,
+            jitter_strength=jitter_strength
+        )
+
+        input_height = self.size_crops[0]  # get global view crop
+        test_transform = transforms.Compose([
+            transforms.Resize(int(input_height + 0.1 * input_height)),
+            transforms.CenterCrop(input_height),
+            self.final_transform,
+        ])
+
+        # replace last transform to eval transform in self.transform list
+        self.transform[-1] = test_transform
+
+
+class SwAVFinetuneTransform(object):
+    def __init__(
+        self,
+        input_height: int = 224,
+        jitter_strength: float = 1.,
+        normalize=None,
+        eval_transform: bool = False
+    ) -> None:
+
+        self.jitter_strength = jitter_strength
+        self.input_height = input_height
+        self.normalize = normalize
+
+        self.color_jitter = transforms.ColorJitter(
+            0.8 * self.jitter_strength,
+            0.8 * self.jitter_strength,
+            0.8 * self.jitter_strength,
+            0.2 * self.jitter_strength
+        )
+
+        if not eval_transform:
+            data_transforms = [
+                transforms.RandomResizedCrop(size=self.input_height),
+                transforms.RandomHorizontalFlip(p=0.5),
+                transforms.RandomApply([self.color_jitter], p=0.8),
+                transforms.RandomGrayscale(p=0.2)
+            ]
+        else:
+            data_transforms = [
+                transforms.Resize(int(self.input_height + 0.1 * self.input_height)),
+                transforms.CenterCrop(self.input_height)
+            ]
+
+        if normalize is None:
+            final_transform = transforms.ToTensor()
+        else:
+            final_transform = transforms.Compose([transforms.ToTensor(), normalize])
+
+        data_transforms.append(final_transform)
+        self.transform = transforms.Compose(data_transforms)
+
+    def __call__(self, sample):
+        return self.transform(sample)
+
+
+class GaussianBlur(object):
+    # Implements Gaussian blur as described in the SimCLR paper
+    def __init__(self, kernel_size, p=0.5, min=0.1, max=2.0):
```

Comment:
Might not want to use `min` and `max` since these are Python built-ins. Could lead to potential issues.